Libplanet/Blocks/BlockHeader.cs

```diff
@@ -68,7 +73,8 @@ namespace Libplanet.Blocks
             BigInteger totalDifficulty,
             ImmutableArray<byte> previousHash,
             ImmutableArray<byte> txHash,
-            ImmutableArray<byte> hash)
+            ImmutableArray<byte> hash,
+            ImmutableArray<byte> actionHash)
```

Comment:
I think this should be added to Backward-incompatible API changes.

Reply:
It changed at 125c70480da14586c337c2e2e883c15a65e860f1